mldsa/fips202/fips202.h

```diff
@@ -64,7 +82,14 @@ __contract__(
 );
 
 #define shake256_squeezeblocks FIPS202_NAMESPACE(shake256_squeezeblocks)
-void shake256_squeezeblocks(uint8_t *out, size_t nblocks, keccak_state *state);
+void shake256_squeezeblocks(uint8_t *out, size_t nblocks, keccak_state *state)
+__contract__(
+  requires(nblocks <= 8 /* somewhat arbitrary bound */)
```

Comment:
This bound should not be required. The proof should be fine for any value of nblocks.
Similarly for keccak_squeezeblocks()